lib/utils.js

```diff
@@ -29,18 +29,53 @@ const path = require('path');
 const electron = require('electron');
 const app = electron.app || electron.remote.app;
 
+// define the defaultDataPath and current directory
+const defaultBasePath = 'storage';
+let currentDirectory = '';
+
+/**
+ * @summary Parse directory to avoid parent directories
+ * @function
+ * @private
+ *
+ * @returns {Strings} parsed user data path without parent directories
+ */
+function getDirectory(directory){
```

Comment:
I like the security thinking here, however I don't think we have any good reason to restrict the settings path to be inside a certain directory. A good use case of this feature is that people can store settings whenever they want, so I guess we can leave out this normalization routine.